Zend/zend_API.c

```diff
@@ -2032,6 +2032,59 @@ static void zend_check_magic_method_args(
 	}
 }
 
+static const char *zend_get_return_type_by_const(int type)
```

Comment:
Isn't the same possible to achieve by converting the mask to `zend_type` by using `ZEND_TYPE_INIT_MASK()`, and then using `zend_type_to_string()`?

Reply:
Yes, going through zend_type_to_string() would be better here.